pki.spec

```diff
@@ -1028,7 +1028,7 @@ BuildArch:        noarch
 Obsoletes:        pki-tests < %{version}-%{release}
 Provides:         pki-tests = %{version}-%{release}
 
-Requires:         python3-pylint
+#Requires:         python3-pylint
```

Comment:
Could you add a conditional so that the python3-pylint dependency remains there in Fedora versions older than 43?